index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://unicons.iconscout.com/release/v4.0.0/css/line.css"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <title>Portfolio</title>
+  </head>
+  <body>
+    <!-- HEADER -->
+    <header>
+      <nav>
+        <a href="#">Prosper</a>
```

Comment:
- Hey, please, kindly move  `<a href="#">Prosper</a>` out of the nav. it is fine for it to be inside the header. let's improve on this to improve our page layout.

Reply:
Removed the nav tags

Reply:
@kessie2862, kindly don't remove the nav tag, just move the ` <a href="#">Prosper</a>` out side the nav tag. 
Do ask questions when you don't seem to understand the suggested changes :fist_raised:

Reply:
Alright thanks. I just made that correction.